test/In.ProjectEKA.FHIRHipTest/DataFlow/CollectTest.cs

```diff
@@ -0,0 +1,82 @@
+using Hl7.Fhir.Utility;
+
+namespace In.ProjectEKA.DefaultHipTest.DataFlow
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using FHIRHip.DataFlow;
+    using FluentAssertions;
+    using HipLibrary.Patient.Model;
+    using Moq;
+    using Moq.Protected;
+    using Optional.Unsafe;
+    using Xunit;
+
+    [Collection("Collect Tests")]
+    public class CollectTest
+    {
+        [Fact]
+        private async void ReturnCallDataFlowService()
```

Comment:
This tests the happy path. Do we have a test case when the URL is not mentioned?

Reply:
There is no point of adding a failure path test as we will be mocking the configuration object